File: decrypt/decrypt.js
const crypto = require('crypto');
const fetch = require('node-fetch');

const key = 'HASH_KEY';
const iv = 'IV_KEY';

fetch('WEBHOOK_QUERY_API')
	.then((res) => res.text())
	.then((dat) => {

		const decipher = crypto.createDecipheriv(
			'AES-128-CBC',
			key,
			iv
		);

		let json = decipher.update(
			dat,
			'base64',
			'utf8'
		);

		json += decipher.final('utf8');

		// user answer json
		console.log(JSON.stringify(JSON.parse(json)));

	})
	.catch((err) => console.log(err));

File: decrypt/decrypt.py
import requests
import base64
from Crypto.Cipher import AES

WEBHOOK_QUERY_API = 'WEBHOOK_QUERY_API'
KEY = b'KEY'
IV = b'IV'

response = requests.get(WEBHOOK_QUERY_API)
data = response.content

encrypted = base64.b64decode(data)
cipher = AES.new(KEY, AES.MODE_CBC, IV)
decrypted_data = cipher.decrypt(encrypted)

# user answer json
print(decrypted_data.decode())